babel.so/src/rvs_stress.cpp

```diff
@@ -70,35 +71,37 @@ void run_stress(std::pair<int, uint16_t> device, int num_times, int ARRAY_SIZE,
 {
   std::string   msg;
   std::streamsize ss = std::cout.precision();
-
+  std::stringstream sstr;
   if (!output_as_csv)
   {
-    std::cout << "Running kernels " << num_times << " times" << std::endl;
+    msg = "Running kernels " + std::to_string(num_times) + " times, " ;
```

Comment:
There is a "," at the end. Do we need it ?

Reply:
yeah to accomodate precision log which comes next, because using logging library we will be adding tag and timestamp for every new log line and i am trying to keep all relevent info in single log line for num of times and what precision